src/main/java/org/fundacionjala/pivotal/objects/EditProject.java

```diff
@@ -0,0 +1,126 @@
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Class for edit Project Page Objects.
+ */
+public class EditProject extends Base {
+
+    //project_description
+    @FindBy(how = How.ID, using = "project_description")
+    private WebElement txtProjectDescription;
+
+    //#project_enable_tasks
+    @FindBy(how = How.CSS, using = "#project_enable_tasks")
+    private WebElement txtTask;
+
+    //#project_public
+    @FindBy(how = How.CSS, using = "#project_public")
+    private WebElement txtPublic;
+
+    //#project_week_start_day
+    @FindBy(how = How.CSS, using = "#project_week_start_day")
+    private WebElement txtProjectWeek;
+
+    //#settings_form  input.save_bar__submit
+    @FindBy(how = How.CSS, using = "#settings_form  input.save_bar__submit")
+    private WebElement saveEditProject;
+
+    @FindBy(how = How.CSS, using = "#save_success_bar div.message")
+    private WebElement messageSaved;
+
+    @FindBy(how = How.CSS, using = "#project_start_date")
+    private WebElement txtDateProject;
+
+    /**
+     * Method for set Date Project.
+     * @param date String.
+     */
+    private void setTxtDateProject(final String date) {
+        CommonMethods.setWebElement(txtDateProject, date);
+    }
+
+    /**
+     * Method for set txt Project week.
+     * @param day String.
+     */
+    private void setTxtProjectWeek(final String day) {
+        CommonMethods.clickWebElement(txtProjectWeek);
+        CommonMethods.setWebElement(txtProjectWeek, day);
+    }
+
+    /**
+     * Method for Message saved.
+     * @return messageSaved.
+     */
+    public String setMessageSaved() {
+        CommonMethods.waitWebElement(messageSaved);
+        return messageSaved.getText();
+    }
+
+    /**
+     * Method for SAve edit Project.
+     */
+    public void setSaveEditProject() {
+        CommonMethods.clickWebElement(saveEditProject);
+    }
+
+    /**
+     * Method for txt Public.
+     * @param check String.
+     */
+    private void setTxtPublic(final String check) {
+        if (check.equalsIgnoreCase("false")) {
+            CommonMethods.clickWebElement(txtPublic);
+        }
+    }
+
+    /**
+     *
+     * Method for txt Project Description.
+     * @param description String.
+     */
+    private void setTxtProjectDescription(final String description) {
+        CommonMethods.setWebElement(txtProjectDescription, description);
+    }
+
+    /**
+     * Method for txt Task.
+     * @param check String.
+     */
+    private void setTxtTask(final String check) {
+        if (check.equalsIgnoreCase("false")) {
+            CommonMethods.clickWebElement(txtTask);
+        }
+    }
+
+    /**
+     * Method for get Strategy Step Map.
+     * @param values Map<DescriptionValues, Object>.
+     * @return strategyMap.
+     */
+    public Map<DescriptionValues, Meth> getStrategyStepMap(final Map<DescriptionValues, Object> values) {
+        Map<DescriptionValues, Meth> strategyMap = new HashMap<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert this Map to an EnumMap. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1640)